src/types/instantsearch.ts

```diff
@@ -64,6 +64,8 @@ export type Hit = {
   __queryID?: string;
 };
 
+export type ResultHit = Omit<Hit, '__position' | '__queryID'>;
```

Comment:
I agree this way of formulating it could be more intuitive.

Reply:
I'll rename the base one `AlgoliaHit` and the InstantSearch one (with `__position`, etc.) `Hit`.

@tkrugg Here we just don't want users to have to pass a hit containing `__position`. If you pass it, it's still fine because it's satisfied with `[key: string]: any;`.

Reply:
Fixed in 0619e161e9a871a8c5035771c11f0a4d02cbdea5.